src/Console/InfectionContainer.php

```diff
@@ -0,0 +1,193 @@
+            return new PathReplacer($c['filesystem'], $c['phpunit.config.dir']);
+        };
+
+        $c['test.framework.factory'] = function (InfectionContainer $c): Factory {
+            return new Factory($c['tmp.dir'], $c['project.dir'], $c['testframework.config.locator'], $c['xml.configuration.helper'], $c['phpunit.junit.file.path'], $c->getInfectionConfig(), $c['version.parser']);
+        };
+
+        $c['xml.configuration.helper'] = function (InfectionContainer $c): XmlConfigurationHelper {
+            return new XmlConfigurationHelper($c['path.replacer']);
+        };
+
+        $c['mutant.creator'] = function (InfectionContainer $c): MutantCreator {
+            return new MutantCreator($c['tmp.dir'], $c['differ'], $c['pretty.printer']);
+        };
+
+        $c['differ'] = function (): Differ {
+            return new Differ(
+                new BaseDiffer()
+            );
+        };
+
+        $c['dispatcher'] = function (): EventDispatcher {
+            return new EventDispatcher();
+        };
+
+        $c['parallel.process.runner'] = function (InfectionContainer $c): ParallelProcessRunner {
+            return new ParallelProcessRunner($c['dispatcher']);
+        };
+
+        $c['testframework.config.locator'] = function (InfectionContainer $c): TestFrameworkConfigLocator {
+            return new TestFrameworkConfigLocator($c['phpunit.config.dir']/*[phpunit.dir, phpspec.dir, ...]*/);
+        };
+
+        $c['diff.colorizer'] = function (): DiffColorizer {
+            return new DiffColorizer();
+        };
+
+        $c['test.file.data.provider.phpunit'] = function (InfectionContainer $c): TestFileDataProvider {
+            return new CachedTestFileDataProvider(
+                new PhpUnitTestFileDataProvider($c['phpunit.junit.file.path'])
+            );
+        };
+
+        $c['version.parser'] = function (): VersionParser {
+            return new VersionParser();
+        };
+
+        $c['lexer'] = function (): Lexer {
+            return new Lexer\Emulative([
+                'usedAttributes' => [
+                    'comments', 'startLine', 'endLine', 'startTokenPos', 'endTokenPos', 'startFilePos', 'endFilePos',
+                ],
+            ]);
+        };
+
+        $c['parser'] = function ($c): Parser {
+            return (new ParserFactory())->create(ParserFactory::PREFER_PHP7, $c['lexer']);
+        };
+
+        $c['pretty.printer'] = function (): Standard {
+            return new Standard();
+        };
+
+        $c['mutators'] = function (InfectionContainer $c): array {
+            $mutatorConfig = $c->getInfectionConfig()->getMutatorsConfiguration();
+
+            return (new MutatorsGenerator($mutatorConfig))->generate();
+        };
+
+        return $c;
+    }
+
+    private function getInfectionConfig(): InfectionConfig
+    {
+        return $this['infection.config'];
+    }
+
+    public static function prepareArgv($argv)
```

Comment:
this method is not related with container. I'm not sure that this is the right place

Reply:
I'm not sure too. That's just how it went. Any suggestions?